.changeset/tender-keys-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wagmi/core": patch
```

Comment:
Adding Wagmi so it pulls in this fix.

```suggestion
"@wagmi/core": patch
"wagmi": patch
```

Reply:
Do I need to add `@wagmi/connectors` as well?

<img width="375" alt="SCR-20240109-jlcf" src="https://github.com/wevm/wagmi/assets/22665058/31037fae-6c12-4616-94c7-7295e79c5bc4">